ERCS/erc-4337.md

```diff
@@ -993,15 +993,15 @@ An array of reputation entries with the fields:
 
 ## Backwards Compatibility
 
-This EIP does not change the consensus layer, so there are no backwards compatibility issues for Ethereum as a whole. Unfortunately it is not easily compatible with pre-[ERC-4337](./eip-4337.md) accounts, because those accounts do not have a `validateUserOp` function. If the account has a function for authorizing a trusted op submitter, then this could be fixed by creating an [ERC-4337](./eip-4337.md) compatible account that re-implements the verification logic as a wrapper and setting it to be the original account's trusted op submitter.
+This EIP does not change the consensus layer, so there are no backwards compatibility issues for Ethereum as a whole. Unfortunately it is not easily compatible with pre-[ERC-4337](./erc-4337.md) accounts, because those accounts do not have a `validateUserOp` function. If the account has a function for authorizing a trusted op submitter, then this could be fixed by creating an [ERC-4337](./erc-4337.md) compatible account that re-implements the verification logic as a wrapper and setting it to be the original account's trusted op submitter.
```

Comment:
afaik, it is enough to reference the actual doc only once, so better to remove the duplications, instead of updating them all.

Reply:
even if it sounds counter-intuitive, you should keep the `./eip-XXX` relative links here, at least for now, as can be seen in issue #86 
PR #42 is supposed to fix all md files in one shot later on, but for now the HTML Proofer run will fail if you put the `./erc-XXX` relative links, and prevent you from getting the PR approved

Reply:
Noted thanks @Joachim-Lebrun! That is counter-intuitive indeed :)